Doc/library/sqlite3.rst

```diff
@@ -2437,9 +2437,9 @@ or if :attr:`~Connection.autocommit` is ``True``,
 the context manager does nothing.
 
 .. note::
-
    The context manager neither implicitly opens a new transaction
-   nor closes the connection.
+   nor closes the connection. If you need a context manager, consider
```

Comment:
Isn't this supposed to be "If you need to close a context manager" instead of "If you need a context manager", I am inferring these semantics from the next wording i.e "using :meth:`contextlib.closing`".

Reply:
Sure, yeah. I think I missed a wording to have this more direct and specific. Referring from @erlend-aasland's suggested change, I missed out "closing context". Thanks for this insight I had not noticed this.